lib/launcher/test/unit/Cardano/LauncherSpec.hs

```diff
@@ -92,10 +94,21 @@ spec = do
         name `shouldBe` "foobar"
 
 -- | A command that will run for a short time then exit with the given status.
```

Comment:
I think this comment is no longer correct. (The given status is no longer passed as a parameter.)

Reply:
OK thanks I have fixed it.